tests/Makefile

```diff
@@ -7,25 +7,48 @@ OPENSHIFT_PASS=
 OPENSHIFT_LOGIN_PROVIDER=
 # Setting SKIP_INSTALL will let you run the tests against an ODH instance that is already setup
 SKIP_INSTALL=
+
+# SKIP_OPERATOR_INSTALL, SKIP_KFDEF_INSTALL will be skipped by setting true for SKIL_INSTALL.
+# However, if you set the first 2 variables to true then the openshift identifier will be configured.
+# Ex)SKIP_INSTALL= , SKIP_OPERATOR_INSTALL=True, SKIP_KFDEF_INSTALL=True
+
+# Setting SKIP_OPERATOR_INSTALL will let you skip deploying ODH Operater.This will enable more delicate manipulation.
+SKIP_OPERATOR_INSTALL=
+
+# Setting SKIP_KFDEF_INSTALL will let you skip creating Kfdef CR. This will enable more delicate manipulation.
+SKIP_KFDEF_INSTALL=
+
 # Setting TESTS_REGEX will allow you to change which tests are going to be run
 TESTS_REGEX=
 # Location inside the container where CI system will retrieve files after a test run
 ARTIFACT_DIR=/tmp/artifacts
 LOCAL_ARTIFACT_DIR="${PWD}/artifacts"
 
+# Setting this for pusing ODH Manifests Test Image to quay.io for ODS Operator Test Harness
+DEFAULT_IMAGE_REGISTRY=quay.io
+DEFAULT_REGISTRY_NAMESPACE=opendatahub
```

Comment:
I think this should probably be in the modh org instead of opendatahub

Reply:
Updated.